etna/records/models.py

```diff
@@ -49,10 +49,10 @@ class Record(DataLayerMixin, APIModel):
     repo_archon_value: str = ""
     level_code: str = ""
     level: str = ""
-    association_reference_number: str = ""
-    association_summary_title: str = ""
+    template_reference_number: str = ""
     template_summary_title: str = ""
-    data_source: str = ""
```

Comment:
I can confirm this not referenced anywhere other than in tests

Reply:
These fields are not in tests